README.md

```diff
@@ -29,6 +37,37 @@ If you would like to join a relayer testnet, please [check out the instructions]
 |-------|--------|----------------|
 | [`gaia`](https://github.com/cosmos/gaia) | ![gaia](https://github.com/cosmos/relayer/workflows/TESTING%20-%20gaia%20to%20gaia%20integration/badge.svg) | `transfer` |
 
+## Features
+
+The relayer supports the following:
+- creating/updating IBC Tendermint light clients
+- creating IBC connections
+- creating IBC transfer channels.
+- initiating a cross chain transfer
+- relaying a cross chain transfer transaction, its acknowledgement, and timeouts
+- relaying from state
+- relaying from streaming events 
+- sending an UpgradePlan proposal for an IBC breaking upgrade
+- upgrading clients after a counterparty chain has performed an upgrade for IBC breaking changes
+
+The relayer currently cannot:
```

Comment:
Also I believe there is also some inflexibility in creating `ClientState`, eg can't change the `UpgradePath` from its default setting. These should also be listed.